src/NodeServices/WorkspaceEvents.cs

```diff
@@ -76,6 +76,20 @@ internal static void OnWorkspaceSettingsChanged(double scaleFactor)
                 handler(new WorkspacesSettingsChangedEventArgs(scaleFactor));
             }
         }
+
+        /// <summary>
+        /// An event raised when workspace EnableLegacyPolyCurveBehavior setting is changed.
+        /// </summary>
+        // TODO: This property should be removed in Dynamo 4.0.
```

Comment:
which property are you referring to in this TODO?

Reply:
I meant the event.